ProjFS.Mac/PrjFSKextLogDaemon/PrjFSKextLogDaemon.cpp

```diff
@@ -155,6 +165,123 @@ static os_log_type_t KextLogLevelAsOSLogType(KextLog_Level level)
     }
 }
 
+static void LogPanics()
+{
+    DIR *dr = opendir(PanicLogDirectory);
+    if (dr == NULL)
```

Comment:
Similarly, the modern C++ way to write this is

```
    if (dr == nullptr)
```

Reply:
Similar comment elsewhere, all `NULL`s should be replaced with `nullptr`